src/math/exp_fcn_hifi4.c

```diff
@@ -0,0 +1,169 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2022 Intel Corporation. All rights reserved.
+//
+// Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+//
+//
+
+#include <sof/math/exp_fcn_hifi4.h>
+#include <sof/common.h>
+#include <stdint.h>
+#include <stdbool.h>
+#include <stddef.h>
+
+#ifdef EXPONENTIAL_HIFI4
+#include <xtensa/tie/xt_hifi4.h>
+
+
+static void mul_s64(ae_int64 in_0, ae_int64 in_1, ae_int64 * ptroutbitshi,
```

Comment:
Extra blanks after ae_int64 *.  You could also avoid this function proto with move of mul_s64() here.